app/views/my/access_token.html.erb

```diff
@@ -0,0 +1,140 @@
+            </th>
+            <th>
+              <div class="generic-table--sort-header-outer">
+                <div class="generic-table--sort-header">
+                  <%= l(:label_access_token_action_header) %>
+                </div>
+              </div>
+            </th>
+          </tr>
+          </thead>
+          <tbody>
+          <% if Setting.feeds_enabled != '0' %>
+              <tr>
+                <td><%= l(:label_feeds_access_key) %></td>
+                <td>
+                  <% if @user.rss_token %>
+                    <span title="<%= format_time(@user.rss_token.created_on) %>">
+                      <%= format_time(@user.rss_token.created_on.to_s) %>
+                    </span>
+                  <% else %>
+                      <%= l(:label_missing_feeds_access_key) %>
+                  <% end %>
+                </td>
+                <td><%= l(:label_access_token_expiration_indefinite) %></td>
+                <td>
+                  <%= link_to l(:button_reset), {:action => 'reset_rss_key'}, :method => :post %>
+                  <a href="" click-notification="<%= l(:notice_value_information, key_name: l(:label_feeds_access_key), value: @user.api_key) %>" click-notification-type="notice">
+                    <%= l(:button_show) %>
+                  </a>
+                </td>
+              </tr>
+          <% end %>
+          <% if Setting.rest_api_enabled != '0' %>
+              <tr>
+                <td><%= l(:label_api_access_key) %></td>
+                <td>
+                  <% if @user.api_token %>
+                    <span title="<%= format_time(@user.api_token.created_on) %>">
+                      <%= format_time(@user.api_token.created_on.to_s) %>
+                    </span>
+                  <% else %>
+                      <%= l(:label_missing_api_access_key) %>
+                  <% end %>
+                </td>
+                <td><%= l(:label_access_token_expiration_indefinite) %></td>
+                <td>
+                  <%= link_to l(:button_reset), {:action => 'reset_api_key'}, :method => :post %>
+                  <a href="" click-notification="<%= l(:notice_value_information, key_name: l(:label_api_access_key), value: @user.api_key) %>" click-notification-type="notice">
+                    <%= l(:button_show) %>
+                  </a>
+                </td>
+              </tr>
+          <% end %>
+          <%= call_hook(:view_access_tokens_table, user: @user) %>
+          </tbody>
+        </table>
+        <div class="generic-table--header-background"></div>
+      </div>
+    </div>
+<% else %>
+  <div class="generic-table--no-results-container" ng-if="!rows.length">
+    <h2 class="generic-table--no-results-title">
+      <i class="icon-info"></i>
+      {{ I18n.t('js.access_tokens.no_results.title') }}
+    </h2>
+    <div class="generic-table--no-results-description"
+         ng-bind-html="I18n.t('js.access_tokens.no_results.description')"></div>
+  </div>
```

Comment:
Why do you use JavaScript here to insert the translations? You could very well do that server-side...